eng/Subsets.props

```diff
@@ -360,9 +360,9 @@
                              $(CoreClrProjectRoot)tools\PdbChecker\PdbChecker.csproj;
                              $(CoreClrProjectRoot)tools\AssemblyChecker\AssemblyChecker.csproj" Category="clr" Condition="'$(DotNetBuildSourceOnly)' != 'true'"/>
     <!-- skip the architectures that don't have LKG runtime packs -->
-    <ProjectToBuild Include="$(CoreClrProjectRoot)tools\aot\crossgen2\crossgen2_publish.csproj" Condition="'$(NativeAotSupported)' == 'true'" Category="clr" />
+    <ProjectToBuild Include="$(CoreClrProjectRoot)tools\aot\crossgen2\crossgen2_publish.csproj" Condition="'$(NativeAotSupported)' == 'true' and '$(StageOneBuild)' != 'true'" Category="clr" />
```

Comment:
@jkoritzinsky `ILCompiler_publish.csproj` is not listed in this file (or anywhere else). Was it an oversight or something for the future?

Reply:
I think there was an intent to hook it up for the `Microsoft.DotNet.ILCompiler` package, but I think it didn't quite happen. Looks like we still publish from the `ILCompiler.csproj` project and ship that. I think the build needs to be adjusted to hook this up correctly (use `ILCompiler_publish.csproj` to publish ilc and remove the publishing logic from `ILCompiler.csproj`)